docling/datamodel/pipeline_options.py

```diff
@@ -41,6 +41,18 @@ class EasyOcrOptions(OcrOptions):
         protected_namespaces=(),
     )
 
+class PaddleOcrOptions(OcrOptions):
+    kind: Literal["paddleocr"] = "paddleocr"
+    lang: str = "en"
```

Comment:
here we should use the same type as the base class which is `List[str]`

Reply:
PaddleOCR doesn't support a list of languages as it's argument. We can use a List of string however, then we will have to hardcode the behavior that we only pick the first value of the provided list. Is that ok? 

From PaddleOCR:
param lang must in dict_keys(['ch', 'en', 'korean', 'japan', 'chinese_cht', 'ta', 'te', 'ka', 'latin', 'arabic', 'cyrillic', 'devanagari'])

Reply:
In the `PaddleOcrOptions`, you can overwrite the type annotation of the `lang` field with type annotations ensuring that the list must have lenght 0 or 1. 
Also, I suggest that the accepted input languages is in sync with what the top-level `paddleocr` API accepts, it is parsed [here](https://github.com/PaddlePaddle/PaddleOCR/blob/main/paddleocr.py#L423).

Reply:
Ok Got it. Also, one small doubt the accepted input language thing, paddle ocr has their own validation checks so do we need to include accepted input languages here in some form wouldn't it be redundant?